ergo-p2p/src/client.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(dead_code)] // TODO: remove
+
+use futures::channel::mpsc;
+use futures::channel::oneshot;
```

Comment:
Out of curiosity, any particular reason we're going with `futures-rs` for these over `tokio`?

Had a bit of a google for comparisons but didn't find that much apart from `futures-rs` being developed by the `rust` team

Reply:
Good catch! No reason apart from it was used in Zebra. Switched to tokio's.